src/test/rustdoc/no-compiler-reexport.rs

```diff
@@ -1,7 +1,7 @@
-// compile-flags: --no-defaults
-
 #![crate_name = "foo"]
 
-// @has 'foo/index.html' '//code' 'extern crate std;'
+// even with --document-hidden-items --document-private-items,
+// foo/index.html never contains `extern crate std;`
```

Comment:
This seems fine to me, it makes the behavior match that the injected re-exports are hidden too. I want to take a look at the git history of the test before merging though.

Reply:
Can you add a `!has` check here to make sure the extern crate statement doesn't appear in the docs?

Reply:
Looks like it was added in https://github.com/rust-lang/rust/pull/77785 - @GuillaumeGomez is there any particular reason you wanted `extern crate std;` to appear when documenting with `--document-hidden-items`? I don't think it's related to that flag in any meaningful way.

Reply:
This is surprising indeed. No I don't remember at all why I would have wanted to have the `extern crate std;` displayed...